### PR TITLE
Add more results. fix search result height for smaller screens

### PR DESCRIPTION
**Pull request by materialsmoke**

### CODE REVIEW

File: assets/js/acf/fields/markdown-editor.js [hunk 1]
```diff
@@ -339,7 +339,7 @@
       languages.forEach(language => {
         if(language.name === currentSelectedLanguageName){
           currentSelectedLanguageDomain = language.home_url;
-          console.log('currentSelectedLanguageDomain', currentSelectedLanguageDomain);
+          // console.log('currentSelectedLanguageDomain', currentSelectedLanguageDomain);
```
Comment (MaltheGram) [inline assets/js/acf/fields/markdown-editor.js:342]: Are you sure you want to leave that comment in there?

File: assets/js/acf/fields/markdown-editor.js [hunk 2]
```diff
@@ -392,7 +392,7 @@
                 </a>
               `);
               
-              console.log('There is no result for this search keyword: ' + searchSimpleMDELinkUrl.val());
+              // console.log('There is no result for this search keyword: ' + searchSimpleMDELinkUrl.val());
```
Comment (MaltheGram) [inline assets/js/acf/fields/markdown-editor.js:395]: Why leave out commented code in there?

**Review by MaltheGram — CHANGES_REQUESTED** [review]

**Review by MaltheGram — APPROVED** [review]

File: assets/js/acf/fields/markdown-editor.js (hunk 2 above)
Comment (materialsmoke) [reply assets/js/acf/fields/markdown-editor.js]: fixed

File: assets/js/acf/fields/markdown-editor.js [hunk 3]
```diff
@@ -376,12 +375,12 @@
...
             style="border-left:1px #ddd solid;border-right:1px #ddd solid;width:calc(100% - 34px);overflow: hidden;"
           >
-          <span style="color:#4caf50">Searching...</span>
+            <span style="color:#4caf50">Searching...</span>
           </a>
         `);
 
         if(currentSelectedLanguageDomain !== ''){
-          fetch(baseDomain(currentSelectedLanguageDomain, false, true, true) + '/wp-json/app/search/query/' + searchSimpleMDELinkUrl.val()).then(res => res.json()).then(data => {
+          fetch(baseDomain(currentSelectedLanguageDomain, false, true, true) + '/wp-json/app/search/query/' + searchSimpleMDELinkUrl.val() + '/per_page/30').then(res => res.json()).then(data => {
```
Comment (GlebOlsen) [inline assets/js/acf/fields/markdown-editor.js:383]: Why add  '/per_page/30' this to the fetch?

**Comment by materialsmoke** [comment]
Because I spoke to Larsse and he wants this 😉

**Review by GlebOlsen — APPROVED** [review]

**Pull request merged**